lib/builders/dataTypes/richTextEditorBuilder/richTextEditorDataTypeBuilder.ts

```diff
@@ -0,0 +1,154 @@
+﻿import {DataTypeBuilder} from "../dataTypeBuilder";
+import {RichTextEditorToolbarBuilder} from "./richTextEditorToolbarBuilder";
+
+export class RichTextEditorDataTypeBuilder extends DataTypeBuilder {
+  richTextEditorToolbarBuilder: RichTextEditorToolbarBuilder;
+  maxImageSize: number;
+  editorModel: any;
```

Comment:
It should be editorMode: string or mode: string as this value is always "Inline" or "Classic"